src/main/kotlin/fm/unit/dao/Reports.kt

```diff
@@ -48,8 +49,17 @@ interface Reports {
         return report_id
     }
 
-
-    // TODO(karsten): join with testsuites.
-    @SqlQuery("SELECT (commit_hash) FROM reports ORDER BY report_id")
-    fun reports(): List<String>
+    @SqlQuery("""
+        SELECT reports.report_id,
+               SUM((xpath('count(//testcase)', payload))[1]::text::integer) AS ts_tests,
+               SUM((xpath('count(//failure)', payload))[1]::text::integer) AS ts_errors
```

Comment:
I wonder if we should use `COUNT(xpath('//testcase'))` instead. At some point we probably want to fetch all errors anyways so this will change.